src/theme/_variables.css

```diff
@@ -1,81 +1,69 @@
+  --ck-font-size-large: $ZD-FONTSIZES-XL;
+  --ck-font-size-extra-large: $ZD-FONTSIZES-XXL;
 
   /* COLORS */
-  --ck-color-base-background: #fff; /* WHITE */
-  --ck-color-base-foreground: #fff; /* WHITE */
-  --ck-color-base-border: #d8dcde; /* GREY-300 */
-  --ck-color-base-action: #038153; /* GREEN-600 */
-  --ck-color-base-focus: #1f73b7; /* BLUE-600 */
-  --ck-color-base-text: #2f3941; /* GREY-800  */
-  --ck-color-base-error: #cc3340; /* RED-600 */
+  --ck-color-base-background: $ZD-PALETTE-WHITE;
+  --ck-color-base-foreground: $ZD-PALETTE-WHITE;
+  --ck-color-base-border: $ZD-PALETTE-GREY-300;
+  --ck-color-base-action: $ZD-PALETTE-GREEN-600;
+  --ck-color-base-focus: $ZD-PALETTE-BLUE-600;
+  --ck-color-base-text: $ZD-PALETTE-GREY-800;
+  --ck-color-base-error: $ZD-PALETTE-RED-600;
   --ck-color-focus-border: var(--ck-color-base-focus);
-  --ck-color-focus-outer-shadow: rgba(31, 115, 183, .35); /* BLUE-600 w/ alpha */
+  --ck-color-focus-outer-shadow: rgba(31, 115, 183, .35); /* BLUE-600 */
 
   /* SPACING */
-  --ck-spacing-unit: 4px;
+  --ck-spacing-unit: $ZD-SPACE-XXS;
   --ck-spacing-extra-large: calc(var(--ck-spacing-unit) * 4);
   --ck-spacing-large: calc(var(--ck-spacing-unit) * 3);
   --ck-spacing-standard: calc(var(--ck-spacing-unit) * 2);
   --ck-spacing-medium: var(--ck-spacing-unit);
   --ck-spacing-small: calc(var(--ck-spacing-unit) * .5);
   --ck-spacing-tiny: calc(var(--ck-spacing-unit) * .25);
-  --ck-spacing-extra-tiny: calc(var(--ck-spacing-unit) * .25);
+  --ck-spacing-extra-tiny: var(--ck-spacing-tiny);
 
   /* BORDERS */
-  --ck-border-radius: var(--ck-spacing-unit);
-  --ck-border-radius-small: var(--ck-spacing-small);
+  --ck-border-radius: $ZD-BORDERRADII-MD;
+  --ck-border-radius-small: $ZD-BORDERRADII-SM;
 
   /* ICONS */
-  --ck-icon-size: var(--ck-spacing-extra-large);
+  --ck-icon-size: $ZD-ICONSIZES-MD;
 
   /* TOOLTIPS */
-  --ck-color-tooltip-background: #03363d; /* KALE-700 */
+  --ck-color-tooltip-background: $ZD-PALETTE-KALE-700;
   --ck-color-tooltip-text: var(--ck-color-base-background);
 
   /* SHADOWS */
   --ck-color-shadow-drop: rgba(23, 73, 77, .15); /* CHROME-600 */
-  --ck-inner-shadow: rgba(31, 115, 183, .35) 0 0 0 3px inset; /* INSET-FOCUS */
+  --ck-inner-shadow: var(--ck-color-focus-outer-shadow) 0 0 0 3px inset;
   --ck-drop-shadow: var(--ck-color-shadow-drop) 0 20px 30px 0;
 
   /* WIDGET */
   --ck-widget-outline-thickness: 1px;
-  --ck-color-widget-hover-border: #fed6a8; /* YELLOW-300 */
 
   /* BUTTONS */
   --ck-color-button-default-background: transparent;
-  --ck-color-button-default-hover-background: rgba(31, 115, 183, .08);
-  --ck-color-button-default-active-background: rgba(31, 115, 183, .2);
-  --ck-color-button-default-active-shadow: transparent; /* TODO NEEDS WORK */
+  --ck-color-button-default-hover-background: rgba(31, 115, 183, .08); /* BLUE-600 */
+  --ck-color-button-default-active-background: rgba(31, 115, 183, .2); /* BLUE-600 */
```

Comment:
`rrggbbaa`?

Reply:
Same issue as above with the 8-digit hex.